google/cloud/sql/connector/connector.py

```diff
@@ -115,6 +116,7 @@ def __init__(
         self._quota_project = quota_project
         self._sqladmin_api_endpoint = sqladmin_api_endpoint
         self._credentials = credentials
+        self._custom_user_agent = custom_user_agent
```

Comment:
I am fine leaving this as `self._custom_user_agent` for clarity since it is an internal

Reply:
I changed it to user agent -- see how that looks.